src/v2/Apps/Artist/Components/MarketStats.tsx

```diff
@@ -0,0 +1,217 @@
+            </Column>
+
+            <Column span={6}>
+              <GridColumns gridRowGap={[2, 0]}>
+                <Column span={[6]}>
+                  <Text variant="text">Sale price</Text>
+
+                  <Text variant="xxl">${formattedAverageValueSold}</Text>
+                </Column>
+
+                <Column span={[6]}>
+                  <Text variant="text">Sale price over estimate</Text>
+
+                  <Text variant="xxl">
+                    <Join separator={<Spacer mr={0.5} />}>
+                      {deltaIcon}
+                      {formattedMedianSaleOverEstimatePercentage}%
+                    </Join>
+                  </Text>
+                </Column>
+              </GridColumns>
+            </Column>
+          </GridColumns>
+        </Column>
+      </GridColumns>
+      <Spacer mt={12} />
+    </>
+  )
+}
+
+export const MarketStatsFragmentContainer = createFragmentContainer(
+  MarketStats,
+  {
+    priceInsightsConnection: graphql`
+      fragment MarketStats_priceInsightsConnection on PriceInsightConnection {
+        edges {
+          node {
+            medium
+            annualLotsSold
+            annualValueSoldCents
+            sellThroughRate
+            medianSaleOverEstimatePercentage
+          }
+        }
+      }
+    `,
+  }
+)
+
+export const MarketStatsQueryRenderer: React.FC<{
+  artistInternalID: string
+  environment: RelayModernEnvironment
+}> = ({ artistInternalID, environment }) => {
+  return (
+    // <QueryRenderer<MarketStatsQuery>
+    <QueryRenderer<any>
+      environment={environment}
+      variables={{ artistInternalID }}
+      query={graphql`
+        query MarketStatsQuery($artistInternalID: ID!) {
+          priceInsightsConnection: priceInsights(
+            artistId: $artistInternalID
+            sort: ANNUAL_VALUE_SOLD_CENTS_DESC
+          ) {
+            ...MarketStats_priceInsightsConnection
+          }
+        }
+      `}
+      render={({ props, error }) => {
+        if (!props?.priceInsightsConnection) {
+          console.error(error)
+          return <></>
+        }
+
+        return (
+          <MarketStatsFragmentContainer
+            priceInsightsConnection={props.priceInsightsConnection}
+          />
+        )
+      }}
+    />
+  )
+}
+
+export const tracks = {
+  tapMarketStatsInfo: () => {},
+  // tapMarketStatsInfo: (): TappedInfoBubbleArgs => ({
+  //   contextModule: ContextModule.auctionResults,
+  //   contextScreenOwnerType: OwnerType.artistAuctionResults,
+  //   subject: "artistMarketStatistics",
+  // }),
+}
+
+export function formatLargeNumber(number: number, decimalPlaces: number = 0) {
+  if (number < 1000) {
+    return number.toString()
+  } else if (number < 1000000) {
+    return `${(number / 1000).toFixed(decimalPlaces)}k`
+  } else if (number < 1000000000) {
+    return `${(number / 1000000).toFixed(decimalPlaces)}M`
+  } else {
+    return `${(number / 1000000000).toFixed(decimalPlaces)}B`
+  }
+}
```

Comment:
Before this component is merged we'll want to add test coverage (which we generally add for all components). Check out the [ArtistApp tests](src/v2/Apps/Artist/__tests__/ArtistApp.jest.tsx) for a recent example of our patterns.

Reply:
Yep, this utility function and the component still need tests. 

Shall I leave `formatLargeNumber` inside of `MarketStats.tsx` or do we have a better place for utility functions like this?

Reply:
As discussed, whatever you feel is best! If something is likely to be shared it can go in a `Routes/MyRoute/Utils` folder, or if you find that it feels more organized there feel free. Generally tho, it's a little bit better to keep stuff close to where it's being called.